tests/test_copies.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import os
+import shutil
+from shallow_backup import _copy_file, _copy_dir
+from constants import Constants
+
+TEST_DIR = 'shallow-backup-test-dir'
```

Comment:
Let's name one of these `TEST_DIR_FOR_BACKUP` and one of them `TEST_DIR_TO_BACKUP` so there's no confusion about which is which.